### PR TITLE
fix URL wrong address

### PR DESCRIPTION
**Pull request by lamsion**
fix DOCS link for Elastos RunTime (RT) - C++

### CODE REVIEW

File: ElastosBountyProgram/front-end/src/module/page/developer/Component.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ export default class extends StandardPage {
                                     </div>
                                     <div className="button-container">
                                         <Button block={matches}>
-                                            <a target="_blank" href="https://github.com/elastos/Elastos.Community/tree/master/ElastosBountyProgram">
+                                            <a target="_blank" href="https://github.com/elastos/Elastos.Community/blob/3640cec31f917725db0a6e7dd686a95ac975b8c6/ElastosBountyProgram/front-end/src/module/page/developer/learn/detail/TheFourPillars/ElastosRuntime/Component.js">
```
Comment (WiktorStarczewski) [inline ElastosBountyProgram/front-end/src/module/page/developer/Component.js]: Why link to this file?

Comment (lamsion) [reply ElastosBountyProgram/front-end/src/module/page/developer/Component.js]: hi Wiktor, I followed the requirements in Pivotal (https://www.pivotaltracker.com/n/projects/2171366/stories/160813575) as below:

> DOCS link for Elastos RunTime (RT) - C++ (on this URL: https://www.cyberrepublic.org/developer) goes to Github EBP (URL: https://github.com/elastos/Elastos.Community/tree/master/ElastosBountyProgram)
It should go to Runtime page of Github:
https://github.com/elastos/Elastos.Community/blob/3640cec31f917725db0a6e7dd686a95ac975b8c6/ElastosBountyProgram/front-end/src/module/page/developer/learn/detail/TheFourPillars/ElastosRuntime/Component.js

Comment (lamsion) [reply ElastosBountyProgram/front-end/src/module/page/developer/Component.js]: @WiktorStarczewski could you discuss with Mike to see what's the right link here? thanks!

**Comment by lamsion** [comment]
Close it till get agreements in requirements.

**Pull request closed without merging**